src/cloud-code/Parse.Cloud.js

```diff
@@ -511,6 +511,16 @@ ParseCloud.onLiveQueryEvent = function (handler) {
   triggers.addLiveQueryEventHandler(handler, Parse.applicationId);
 };
 
+ParseCloud.afterLiveQueryEvent = function (parseClass, handler) {
```

Comment:
Can you add documentation?

Reply:
Can do. I will also add to the docs repo once merged.